resources/views/groups/edit.blade.php

```diff
@@ -4,7 +4,7 @@
     'helpTitle' => trans('admin/groups/general.about_groups_title'),
     'helpText' => trans('admin/groups/general.about_groups_text'),
     'item' => $group,
-    'formAction' => ($group) ? route('groups.update', ['accessory' => $group->id]) : route('groups.store'),
+    'formAction' => ($group->id) ? route('groups.update', ['accessory' => $group->id]) : route('groups.store'),
```

Comment:
`$group` alone always evaluated `true`, so the form never loaded the `groups.update` route, causing an exception when ->id was not set. What do you think about a combined condition, like `($group !== null && $group->id !== null)` ?